src/south/ProtocolHandler.class.js

```diff
@@ -47,7 +47,7 @@ class ProtocolHandler {
 
     const { logParameters } = this.dataSource
     this.logger = new Logger()
-    this.logger.changeParameters(this.engineConfig.logParameters, logParameters)
+    this.logger.changeParameters(this.engineConfig.engineName, this.engineConfig.logParameters, logParameters)
```

Comment:
maybe we should only send the engineConfig and it's up to the called function to pick engineName and logParameters.